### DISCUSSION TITLE
AlDente does not start me in the menu bar

### DISCUSSION THREAD

**Discussion by apg0108** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1067] [General]
When I turn on the Mac (after updating to macOS Sonoma) I can't see the AlDente battery icon in the menu bar, if I try to open the application manually nothing happens, only when I restart the Mac does it solve it. But it repeats again when I turn it off again.

**Comment by MatthiasKerbl** [comment]
Hi @apg0108,

There is already an issue open for this bug here #1046. We will give updates about this there.

Best,
Matthias

**Reply by apg0108** [reply]
Thanks you!
